backend/src/main/java/com/bounswe2024group10/Tradeverse/dto/portfolio/CreatePortfolioRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.bounswe2024group10.Tradeverse.dto.portfolio;
+
+public class CreatePortfolioRequest {
+    private String username;
+    private String name;
+    private String visibility;
```

Comment:
Where is the amount variable?

Reply:
Added as BigDecimal, but I am not sure whether using BigDecimal is okey.